### PR TITLE
chore: add supported kubernetes versions

### PR DESCRIPTION
**Pull request by supereagle**
**What this PR does / why we need it**:

Add supported Kubernetes versions.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @zhujian7 @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ With Cyclone, users end up with the flexibility of workflow orchestration and th
 
 ## Quick Start
 
+> Cyclone has been tested with Kubernetes 1.12, 1.13 and 1.14.
+
 Make sure [Helm](https://helm.sh/) with a version higher than **2.10** is installed ([install guide](https://helm.sh/docs/using_helm/#install-helm)), then install Cyclone with:
```
Comment (zhujian7) [inline README.md]: > a version higher than **2.10**

[Cyclone only officially supports Helm v2.10+, still not supports Helm v3.0.0-alpha.1](https://github.com/caicloud/cyclone/issues/1178#issuecomment-508916939)? It is better to mention tested helm version here?

Comment (supereagle) [reply README.md]: I think there is no need as v3.0.0-alpha.1 is just a pre-release, Helm v3.0.0 is not officially released.

**Pull request merged**